src/test/java/jenkins/bouncycastle/EncodingDecodingTest.java

```diff
@@ -54,7 +56,12 @@
 
     @BeforeClass
     public static void setUpBC() {
-        Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
+        Security.addProvider(new BouncyCastleProvider());
+    }
+
+    @AfterClass
```

Comment:
technically not requires as surefire is configured to not reuse forks, but this is good for IDEs